oak_attestation_verification/build.rs

```diff
@@ -26,12 +26,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     // Tell cargo to rerun this build script if the proto file has changed.
     // https://doc.rust-lang.org/cargo/reference/build-scripts.html#cargorerun-if-changedpath
     for proto_path in proto_paths.iter() {
-        let file_path = std::path::Path::new(proto_path);
-        println!(
-            "cargo:rerun-if-changed={}{}",
-            env!("WORKSPACE_ROOT"),
-            file_path.display()
-        );
+        println!("cargo:rerun-if-changed=../{}", proto_path);
```

Comment:
As a matter of fact, I think the actual paths above should include that prefix instead, see https://github.com/project-oak/oak/blob/main/oak_containers_sdk%2Fbuild.rs#L22 . Does that work?

Reply:
It compiles successfully, but the automatic change detection doesn't infer that changes to files in a different package should cause rebuilds. oak_containers_sdk works because `generate_grpc_code` emits the `rerun-if-changed` directives itself: https://github.com/project-oak/oak/blob/main/oak_grpc_utils/src/lib.rs#L70.

`prost_build::compile_protos`, on the other hand, doesn't emit `rerun-if-changed`: https://github.com/tokio-rs/prost/blob/63c00248abef84337c1010b6a10f3258b2db8b54/prost-build/src/lib.rs#L1071. And without it, cargo only reruns if files change within the crate root, not outside it: https://doc.rust-lang.org/cargo/reference/build-scripts.html#change-detection.

Reply:
yes I am aware the change detection wouldn't work automatically in that case, but could you keep the manual cargo:rerun-if-changed lines here, in combination with fixing the paths above? Basically just move the `..` from this line to the paths above, but leave all the logic as is. Or please feel free to submit as is and I am happy to try things out separately.

Reply:
Sorry, I misunderstood your suggestion. Yup, it works to include the "../" paths above. (I'd assumed the benefit would be that `prost_build::compile_protos` would auto-generate the necessary `rerun-if-changed` directives if the paths were changed, but just being consistent and not needing to add "../" below is useful.)